src/main/resources/templates/trips/trip.html

```diff
@@ -7,18 +7,21 @@
     <br>
     <br>
 
-    <div class="itinerary-box">
-        <h3 th:text="${trip.tripName}">Itinerary</h3>
+    <div class="itinerary-box trip card" id="trip-card">
+        <h3 id="trip-name" th:text="${trip.tripName}"></h3>
+        <br>
         <div class="itinerary-stops">
-            <h5>Starting: </h5>
+            <h5>🚗️ Starting: </h5>
             <p th:text="${trip.startingLocation}"></p>
             <br>
             <h5>Stops: </h5>
-            <p th:text="${trip.stopOne}"></p>
-            <p th:text="${trip.stopTwo}"></p>
-            <p th:text="${trip.stopThree}"></p>
+            <ul>
+                <li th:text="${trip.stopOne}"></li>
+                <li th:text="${trip.stopTwo}"></li>
```

Comment:
I think it every time I see it: the little car at the starting location is a super cute idea.

Is it possible to apply and show the <li> tag only if the stop actually has value? I don't know whether it's possible to do a th:if statement or the like so that there are not empty bullet points in the card if someone doesn't fill in all three slots. Just a thought.

Reply:
fixed!